blueprint/api.apib

```diff
@@ -1358,6 +1358,153 @@ The Stripe plan resource allows users to select a monthly donation amount for a
 
     + Attributes (Record Not Found Response)
 
+# Group Stripe Subscriptions
+
+This endpoint is for creating, editing and returning Stripe Subscriptions.
+
+Stripe Subscriptions allow us to charge a customer's card on a recurring basis. A subscription ties a customer to a particular plan.
+
+## Stripe Subscriptions [/stripe-subscriptions]
+
+### Create a Stripe subscription [POST]
+
++ Parameters
+
+    + id (number) - Id of a Stripe subscription.
```

Comment:
I actually don't think parameters is what we want here. You're probably getting a warning for this.